qlib/tests/data.py

```diff
@@ -70,15 +96,28 @@ def qlib_data(
             data freq, value from [1d], by default 1d
         region: str
             data region, value from [cn, us], by default cn
+        delete_old: bool
+            delete an existing directory, by default True
 
         Examples
         ---------
         python get_data.py qlib_data --name qlib_data --target_dir ~/.qlib/qlib_data/cn_data --version latest --interval 1d --region cn
         -------
 
         """
-        file_name = f"{name}_{region.lower()}_{interval.lower()}_{version}.zip"
-        self._download_data(file_name.lower(), target_dir)
+        _version = re.search(r"(\d+)(.)(\d+)(.)(\d+)", qlib.__version__)[0]
+        if p_version.parse(_version) in SpecifierSet("<=0.6.1"):
+            qlib_version = "0.6.1"
+        else:
+            qlib_version = "latest"
+        file_name = self.QLIB_DATA_NAME.format(
+            dataset_name=name,
+            region=region.lower(),
+            interval=interval.lower(),
+            qlib_version=qlib_version,
```

Comment:
Did you consider the situation that the qlib version changes but the data version remains the same?